### ISSUE TITLE
enhanced preview doesn't handle partial word italics

### ISSUE THREAD

**Issue by af4jm** [dendronhq/dendron#538]
## Please select the area your Bug applys to. (Multiple selections are Possible)

- [ ] UI
- [ ] Workbench
- [ ] Publishing
- [ ] Pods
- [x] Markdown
- [ ] Hierarchy
- [ ] Graph
- [ ] Misc (Choose this if your not sure)

## Describe the bug

When using _ for italics, Dendron markdown preview does not recognize italics for partial words

## To Reproduce

type into any markdown document: `_w_ord *w*ord _word_`

## Expected behavior

italics 3 times

Actual behavior: the first one shows the underbars and is not italic

## Additional context

latest VSCode w/ latest Dendron

## Dendron Log file

Please attach the output of `>Dev:Dendron: Open Logs` here

**Comment by kpathakota** [comment]
Yeah, good catch! The preview definitely needs work. We can try fixing this once we have our own custom parser but I don't expect it to get done until #643 gets done.
